en/advanced_config/esc_calibration.md

```diff
@@ -24,33 +47,90 @@ To calibrate the ESCs:
-:::
+   - The minimum value for a motor (default: `1100us`) should make the motor spin slowly but reliably, and also spin up reliably after it was stopped.
+   
+     You can confirm that the value spins at minimum (still without propellers) in [Actuator Testing](../config/actuators.md#actuator-testing), by moving the test output slider to the first snap position from the bottom.
+
+     To find the "optimal" minimum value reduce the PWM minimum value for the output to the disarmed value (1000us), and increase it in small increments (e.g. 1025us, 1050us, etc) until the motor starts to spin.
+     The correct value should make the motor spin immediately and reliably as you move the slider from disarmed to minimum.
+     It is better to be a little too high than a little too low.
+
+   - The maximum value for a motor (default: `1900us`) should be chosen such that if the value is further increased from there the motor doesn't actually spin any faster.
+
+     You can confirm that the motor spins quickly at the maximum setting in [Actuator Testing](../config/actuators.md#actuator-testing), by moving the associated test output slider to the top snap position.
+
+     To find the "optimal" maximum value, increase the PWM maximum value for the output in increments (e.g. 1925us, 1950us, etc) and listen to determine whether the speed increases.
+     Stop at the increment before the last speed increase.
+
+   - The disarmed value for a motor (default: `1000us`) should make the motor stop and stay stopped.
+
+     You can confirm this in [Actuator Testing](../config/actuators.md#actuator-testing) by moving the test output slider to the snap position at the bottom of the slider and observing that the motor does not spin.
+
+     If the ESC spins with the default value of 1000us then the ESC is not properly calibrated.
+     If using an ESC that can't be calibrated, you should reduce the PWM output value for the output to below where the ESC does not split (such as 900us).
+
+   :::note
+   VTOL and fixed-wing motors do not need any special PWM configuration.
+   With the default PWM configuration they will automatically stop during flight when commanded by the autopilot.
+   :::
+
+## Troubleshooting
+
+
+1. Calibration can state that it has succeeded even when it has failed.
+
+   This happens happen if you do not power the ESC at the right time, or the ESCs don't support calibration.
+   This occurse because PX4 has has no way to know whether or not calibration was successful.
+
+1. Calibration cannot be started if you have a power module configured and connected (for safety reasons).
+
+   Unplug power to the ESCs first.
+   If you're blocked because a power module is necessary to keep your flight controller alive, but you can (un)power the ESCs separately, you can disable the motor just for the ESC calibration using parameters such as [PWM_AUX_MINn](../advanced_config/parameter_reference.md#PWM_AUX_MIN1).

```

Comment:
Are you saying that in order to calibrate the motor must be unpowered, and you can do this by setting the minimum value to (all) motors to some low value - e.g. PWM_AUX_MIN1 to 900us?

FYI, I updated all these items to state "problem first". This usually a better approach.

Reply:
It's a copy paste mistake on my side. I copied the parameter link but didn't change to the correct name 🤦‍♂️ 
Thanks for pointing it out it only occurred to me now when reading your very helpful improvements.